README.adoc

```diff
@@ -127,12 +140,15 @@ followed by fields that correspond to metadata fields.
 *The `description` field is optional and describes the purpose of the metric
 annotation.
 
-The `@Counted` annotation applies to the `list()` method. This annotation counts how many times you request the list of systems methods by visiting `http://localhost:9080/inventory/systems` to view the current contents of the
-inventory.
-* The `absolute` field makes the name of this annotation take the name of the method.
+The `@Counted` annotation applies to the `list()` method. This annotation counts how many times
+you request the list of systems methods by visiting `http://localhost:9080/inventory/systems` to
+view the current contents of the inventory.
+* The `absolute` field gives this annotation the name of the method.
 * The `description` field is optional and describes the purpose of the metric annotation.
-* The default value of the `monotonic` field is `false`. When `false`,
-the annotation counts the current invocations of the annotated method, and the counter is incremented before the annotated method returns. However, set the value to `true` instead.
+* The `monotonic` field is set to `true` so that the counter will only increment.
+
+The `@Counted` and `@Timed` annotations can be also placed in the Resource as
```

Comment:
To emphasize this as a side note, maybe highlight this as an Asciidoc note.

`NOTE: The @Counted and @Timed annotations can be also placed...`

Reply:
@sishida @evelinec can we do that?

Reply:
`NOTE:` is an asciidoc admonition so it should generate an icon.
See http://www.methods.co.nz/asciidoc/userguide.html#X28

Reply:
Yes Yes I know ;). But it is just we are following a structure or a style that is similar to other guides

Reply:
I am fine with adding notes, but what kind of icon will generate as a result of using this asciidoc admonition? Do we need to get an icon from the design team?

Reply:
I talked to @evelinec  we do not need to use ascii note, we can just state it in a sentence starting with something like note ....